docs/quickstart.md

```diff
@@ -1,70 +1,163 @@
+	https://auth.carbonrelay.io/authorize?...
 
-For more details, see [the installation guide](install.md).
+You are now logged in.
+```
 
-## Initialize the Red Sky Ops Manager
+When you initialize your cluster for Red Sky Ops, the following resources will be created:
+- `redsky-system` namespace
+- `experiment` and `trial` custom resource definitions
+- `ClusterRole` and `ClusterRoleBinding` for the controller service account
+- `redsky-controller-manager` deployment
+- an authorization secret for the controller
 
-Once you have the Red Sky Ops Tool you can initialize the manager in your cluster:
+Initialize your cluster with the following command:
 
 ```sh
-$ redskyctl init
+$ redskyctl init --wait
+```
+You should see output similar to this:
+```sh
+namespace/redsky-system created
+customresourcedefinition.apiextensions.k8s.io/experiments.redskyops.dev created
+customresourcedefinition.apiextensions.k8s.io/trials.redskyops.dev created
+clusterrole.rbac.authorization.k8s.io/redsky-manager-role created
+clusterrolebinding.rbac.authorization.k8s.io/redsky-manager-rolebinding created
+deployment.apps/redsky-controller-manager created
+clusterrole.rbac.authorization.k8s.io/redsky-patching-role created
+clusterrolebinding.rbac.authorization.k8s.io/redsky-patching-rolebinding created
+secret/redsky-manager created
 ```
 
-## Create a Simple Experiment
-
-Generally you will want to write your own experiments to run trials on your own applications. For the purposes of this guide we can use the postgres example found in the `redskyops-recipes` [repository on GitHub](https://github.com/redskyops/redskyops-recipes/tree/master/postgres):
+Verify manager is running:
 
 ```sh
-$ kustomize build github.com/redskyops/redskyops-recipes/postgres | kubectl apply -f -
+$ kubectl wait --for condition=Ready=true po \
+  -n redsky-system \
+  -l app.kubernetes.io/name=redskyops
+```
+You should see output similar to this:
+```sh
+pod/redsky-controller-manager-5fb9f4cd4d-g2rn5 condition met
 ```
 
-## Run a Trial
+## Create the Experiment
 
-With your experiment created, you can be begin running trials by suggesting parameter assignments locally. Each trial will create one or more Kubernetes jobs and will conclude by collecting a small number of metric values indicative of the performance for the trial.
+We'll use the Postgres example [here][postgres-example].
+This example will deploy the Postgres application and configure an experiment to tune the memory and CPU limits for Postgres.
+The controller will schedule trials (Kubernetes jobs) using `pgbench` to generate load against our Postgres instance.
+Each trial will test a different set of parameters provided by the Red Sky Ops ML servers.
+The effectiveness of each trial is gauged by the metrics, in this case we contrast cost versus duration.
 
+Deploy the Postgres application and experiment using the following:
+```sh
+$ kustomize build github.com/redskyops/redskyops-recipes/postgres | \
+  kubectl apply -f -
```

Comment:
To do line continuation in these Markdown blocks you need to use `>` on the subsequent line. Subjectively, indentation helps.

For example `$ kustomize ... \` is on one line and `>   kubectl ...` is on the next (that's three spaces after the gt, it's tough because this comment itself is Markdown).

Reply:
These are inside code blocks, so I don't believe `>` is needed. ( check out the preview, they look appropriate )

I know the shell typically inserts `>` to denote being on a newline, but from a usability perspective it prevents a user from just selecting and copying/pasting the command into the shell.

Reply:
Yes, but these are meant to be like screenshots of a console so if the "$" prompt is there, the ">" continuation prompt should be to. This was all stuff I didn't do well initially, but we should fix it now.

I didn't check in Hugo, but:

```console
$ echo \
>   test
test
```

vs.

```console
$ echo \
  test
test
```

Reply:
vs.

```console
$ echo \
> test
test
```

Reply:
Adding the `>` inside the code block breaks the ability for people to copy/paste.

If we take out the line breaks, then you need to scroll horizontally to see the entire command which isn't great either. 

Until we add in some js or something to the theme to provide a copy to clipboard option that can strip the extra characters automatically, I'd really like to not include the `>`.